lldb/include/lldb/Target/MemoryRegionInfo.h

```diff
@@ -141,6 +142,35 @@ class MemoryRegionInfo {
     m_dirty_pages = std::move(pagelist);
   }
 
+  std::string Dump() const {
```

Comment:
Hey @JDevlieghere is there a better area to print MemoryRegionInfo? I must be reinventing the wheel but there isn't an existing method on MemoryRegion?